src/backend/t/0015-BSSched-EventSource-Directory.t

```diff
@@ -257,7 +257,7 @@ push(@files_to_remove,"$ev_dir/$file_name");
 # CLEANUP
 unshift(@files_to_remove,"$eventdir_base/$arch/$evname");
 
-map { ( -d $_ ) ? rmdir $_ : unlink $_ ; die "$_: $!" if $!;} @files_to_remove;
+map { ( -d $_ ) ? rmdir $_ : unlink $_ ; warn "$_: $!" if $!;} @files_to_remove;
```

Comment:
Wow, now that's a horrible line. Please don't use map if you're really looking for a for() loop. And don't use multiple statements in a single line...
```
for (@files_to_remove) {
  multiple lines of your code here
}
```